source-postgres/backfill.go

```diff
@@ -94,25 +118,36 @@ func (db *postgresDatabase) WatermarksTable() string {
 	return db.config.Advanced.WatermarksTable
 }
 
-func (db *postgresDatabase) buildScanQuery(start bool, keyColumns []string, schemaName, tableName string) string {
-	// Construct strings like `(foo, bar, baz)` and `($1, $2, $3)` for use in the query
-	var pkey, args string
+// The set of MySQL column types for which we need to specify `COLLATE "C"` to get
```

Comment:
In the comment, perhaps "MySQL" should be "Postgres"

Reply:
I swear, copy-pasting that "do we need some string key comparison" logic introduced no less than 3 distinct flaws. I should have just typed it out by hand the second time :/

Fixed